libkbfs/key_bundle_v3.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"encoding"
+
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-codec/codec"
+)
+
+// TLFWriterKeyBundleV3 is a bundle of writer keys and historic symmetric encryption
+// keys for a top-level folder.
+type TLFWriterKeyBundleV3 struct {
```

Comment:
What about `TLFReaderKeyBundleV3`? If there's no changes, maybe it could be an alias for `TLFReaderKeyBundleV2` or something, with appropriate comments